widgets/testimonial/styles/default.less

```diff
@@ -1,6 +1,5 @@
-@import "../../../base/less/mixins";
```

Comment:
Yeah, I was just busy writing a larger comment about that. :smile: So we've basically hardcoded the relative paths in there, which is somewhat hacky, but doesn't cause any issues because the relative path in the `@import` is stripped. I think we should be able to, for example, use a command line LESS compiler on this file and if you do that without the relative path it will throw an error. And then, as you mentioned, the fact that IDEs like PHPStorm and IntelliJ complain.